docker/dockerfiles/be/cn_entrypoint.sh

```diff
@@ -173,3 +173,12 @@ if [[ "x$LOG_CONSOLE" == "x1" ]] ; then
     addition_args="--logconsole"
 fi
 $STARROCKS_HOME/bin/start_cn.sh $addition_args
+ret=$?
+if [[ $ret -ne 0 && "x$LOG_CONSOLE" != "x1" ]] ; then
+    nol=50
+    log_stderr "Last $nol lines of cn.INFO ..."
+    tail -n $nol $STARROCKS_HOME/log/cn.INFO
+    log_stderr "Last $nol lines of cn.out ..."
+    tail -n $nol $STARROCKS_HOME/log/cn.out
```

Comment:
I checked a k8s env, and there is no `$STARROCKS_HOME` environment.

Reply:
it is set in entrypoint